book/src/user/run.md

```diff
@@ -11,6 +11,10 @@ structure, and documentation for all of the config options can be found
 
 * `zebrad start` starts a full node.
 
+## Supported versions
+
+Always make sure your version is under support and that you upgrade regularly to avoid halting. [More information](https://github.com/ZcashFoundation/zebra/blob/main/book/src/dev/release-process.md#supported-releases).
```

Comment:
"Under support" sounds a lot like "unsupported", but I don't think that's what you meant here.

```suggestion
Always run a supported version of Zebra, and upgrade it regularly, so it doesn't become unsupported and halt. [More information](https://github.com/ZcashFoundation/zebra/blob/main/book/src/dev/release-process.md#supported-releases).
```